README.md

```diff
@@ -51,7 +51,7 @@ exports.handler = lumigo.trace(myHandler)
 // typescript
 import lumigo from '@lumigo/tracer';
 
-const tracer = lumigo({ token: 'YOUR-TOKEN-HERE' });
```

Comment:
Same

Reply:
That's what I've added in the `Connect Your Lumigo Account` section though right?